apps/timetable/ui/js/index.js

```diff
@@ -154,9 +157,13 @@ define(['gh.core', 'gh.subheader', 'gh.calendar', 'gh.student-listview', 'jquery
      */
     var setUpIndex = function() {
         addBinding();
-        setUpHeader();
-        setUpCalendar();
         renderLoginModal();
```

Comment:
Micro optimisation alert: We can add this as the last function executed in the callback of `fetchTriposData` is this wouldn't be triggered without the actual data being rendered in the sidebar and therefor shouldn't 'hold up' fetching it.